sdk/tests/iot/provisioning_v2/test_az_iot_proviosining_v2_client_payload.c

```diff
@@ -0,0 +1,204 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I noticed that both the names of the v1 and v2 versions of this file spell provisioning wrong, not related to the current changes but might be good to address.

Reply:
Ouch... we didn't catch that in main. I will make the correction here.